src/applications/ask-va/sass/ask-va-too.scss

```diff
@@ -314,8 +314,8 @@ $facility-locator-shadow: rgba(0, 0, 0, 0.5);
 
 .ava-progress-bar-inner {
   height: 100%;
-  background-color: $color-orange;
-  border: 1px solid $color-orange;
+  background-color: var(--uswds-system-color-orange-40);
+  border: 1px solid var(--uswds-system-color-orange-40);
```

Comment:
No DS semantic token equivalent for this currently so using a primitive uswds token instead.